src/base/static/components/organisms/story-navigator.js

```diff
@@ -29,10 +29,7 @@ class StoryNavigator extends Component {
     };
   }
 
-  async componentDidMount() {
-    // TODO(luke): When AppView is ported to a react component, await
-    // the place collections to load in the App component:
-    await this.props.placeCollectionsPromise;
+  componentDidMount() {
```

Comment:
Our placeCollectionsPromise is now being awaited in AppView, so that we can update our Redux store with our Place POJO's. 

And I think it is better if StoryNavigator isn't concerned with this api call anyway.